### ISSUE TITLE
Higher-taxonomy lookup for TPL-known hybrids

### ISSUE THREAD

**Issue by rossmounce** [wcornwell/taxonlookup#23]
Not high-priority but...

I would like to be able to lookup the higher-level taxonomy of names that are Hybrids, that TPL has data for e.g. × _Elyhordeum chatangensis_ (Roshev.) Tzvelev http://www.theplantlist.org/tpl1.1/record/kew-387028 (Accepted).

It seems like this should be possible, if TPL has data on the name

```
> plant_lookup_version_current()
[1] "1.1.1"
> lookup_table(c("Pinus","× Elyhordeum chatangensis"),missing_action="NA")
  genus   family   order       group
1 Pinus Pinaceae Pinales Gymnosperms
2     ×     <NA>    <NA>        <NA>
> lookup_table(c("Pinus","×Elyhordeum chatangensis"),missing_action="NA")
        genus   family   order       group
1       Pinus Pinaceae Pinales Gymnosperms
2 ×Elyhordeum     <NA>    <NA>        <NA>
> lookup_table(c("Pinus","Elyhordeum chatangensis"),missing_action="NA")
       genus   family   order       group
1      Pinus Pinaceae Pinales Gymnosperms
2 Elyhordeum     <NA>    <NA>        <NA>
```

Desired output/behaviour:

```
> lookup_table(c("Pinus","× Elyhordeum chatangensis"),missing_action="NA")
  genus   family   order       group
1 Pinus Pinaceae Pinales Gymnosperms
2 × Elyhordeum Poaceae Poales Angiosperms
```

**Comment by wcornwell** [comment]
Yeah, we discussed this in depth at one point.  The botanical code is a bit vague on fertile hybrid species  versus infertile hybrid crosses.  And the systematists at our meeting seemed convinced that the majority (although not all) of the names with the "x" notation in the plant list were infertile or most infertile hybrids. 

x Elyhordeum seems like it's not an exception: see http://www.herbarium.usu.edu/treatments/xElyhordeum.htm

So the folks studying diversity and diversification wanted them excluded from the species counts. It would be a bit tricky to keep them in the lookup but drop them from the species counts.  Hmmm

**Comment by rossmounce** [comment]
Understood. 
It's not a biggie for me. Just thought I'd mention it :smile: 

For my use case, I just want to know what the plant record is (high-level taxonomy-wise), regardless of onward fertility. I can see how/why different use cases wouldn't want this info included.
